packages/core/src/components/Tabs/Tab.jsx

```diff
@@ -33,12 +33,13 @@ export class Tab extends React.PureComponent {
     return (
       <a
         aria-selected={String(this.props.selected)}
+        aria-disabled={String(this.props.disabled)}
```

Comment:
What's the visual styling of this in the app you're using this in? I think we'll want some visual indication that the tab is disabled. 

I'm also, mainly out of curiosity, interested to learn more about the use case you're using this for? This would help inform the guidance that goes alongside the component.

Reply:
Since we switched qpp-style to use this repo instead of react-tabs, we wanted to not break the support for disabled tabs:
<img width="794" alt="screen shot 2018-06-25 at 3 30 02 pm" src="https://user-images.githubusercontent.com/6475104/41871397-c8a88bd2-788c-11e8-81a8-67d937cae19e.png">

Reply:
Could we add similar visual styling into this? We could use the same styling as the [disabled button](https://design.cms.gov/example/components.button.disabled/). This would also probably save y'all from needing to write as much custom styling. Let me know if you'd like me to jump in for the visuals here.